src/main/io/ledstrip.c

```diff
@@ -1055,16 +1055,14 @@ bool isOverlayTypeUsed(ledOverlayId_e overlayType)
 
 void updateRequiredOverlay(void)
 {
-    for (int timID = 0; timID < timTimerCount; timID++) {
-        requiredTimerLayer[timID] = true;
-    }
-    requiredTimerLayer[timBlink] = isOverlayTypeUsed(LED_OVERLAY_BLINK);
-    requiredTimerLayer[timLarson] = isOverlayTypeUsed(LED_OVERLAY_LARSON_SCANNER);
-    requiredTimerLayer[timWarning] = isOverlayTypeUsed(LED_OVERLAY_WARNING);
+    disabledTimerMask = 0;
+    disabledTimerMask |= !isOverlayTypeUsed(LED_OVERLAY_BLINK) << timBlink;
```

Comment:
```
#define TEST_TIM(overlay, tim) do { disabledTimerMask |= !isOverlayTypeUsed(overlay) << (tim); } while (0)
```
And maybe use `if` in the macro ...

Reply:
Well only a few of the timers are candidates for being disabled so I'd have to enumerate them in `isOverlayTypeUsed()` anyway.  So I'm not sure how much will be gained here.  I also have a  new feature PR coming that adds a "blink only when armed" overlay that shares the `timBlink` and `applyLedBlinkLayer()` for efficiency but has a different overlay code `LED_OVERLAY_BLINK_ARMED` so it changes to:

`    requiredTimerLayer[timBlink] = isOverlayTypeUsed(LED_OVERLAY_BLINK) || isOverlayTypeUsed(LED_OVERLAY_BLINK_ARMED);`

Reply:
Yep, probably not worth the complications ... it would be easier if timers were enabled overlay-by-overlay (duplicate test line), but new timers would need enabling somehow ...

The macro is nice when you want to change underlying implementation - just change macro, mapping code stays the same